src/commands/misc/birb.js

```diff
@@ -0,0 +1,19 @@
+const { Command, SwitchbladeEmbed } = require('../../')
+const snekfetch = require('snekfetch')
+
+module.exports = class Birb extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'birb'
```

Comment:
missing this.category

```suggestion
    this.name = 'birb'
    this.category = 'misc'
```